script.js

```diff
@@ -1,4 +1,302 @@
+// ****** How the programme works ******
+//
+// Number of players: 2
+// Number of dice rolled: 2 (default)
+// Game mode: highest combination (default)
+// Players can change how many dice are rolled by keying in number from 1-10 (inclusive)
+// Programme autocombines the numbers according to the game mode
+// Players can change game mode by entering "highest" or "lowest"
+//
+// *************************************
+
+// ///// ******* Global Variables ******* /////
+var gameMode = "highest";
+var player = "Player 1";
+
+var numDiceToRoll = 2;
+
+var p1diceNums = [];
+var p2diceNums = [];
+var p1CombinedNumber = 0;
+var p2CombinedNumber = 0;
+
+var p1score = 0;
+var p2score = 0;
+
+// ///// ********** Functions ********** /////
+
+// ----------- Functions to change global variables -----------
+// Function to change player
+var changePlayer = function () {
+  if (player == "Player 1") {
+    player = "Player 2";
+  } else {
+    player = "Player 1";
+  }
+};
+
+// Function to alternate between game modes
+var changeGameMode = function (input) {
+  if (input == "lowest") {
+    gameMode = "lowest";
+    resetGame();
+    return `
+    Initiating new game rules: the player with the <b>lowest</b> combined number wins! <br><br>
+    The score has been reset.<br><br>
+    Player 1, start by hitting submit.
+    `;
+  } else if (input == "highest") {
+    gameMode = "highest";
+    resetGame();
+    return `
+    Initiating new game rules: the player with the <b>highest</b> combined number wins! <br><br>
+    The score has been reset.<br><br>
+    Player 1, start by hitting submit.
+    `;
+  }
+};
```

Comment:
You could leave the return statement until after the if and else if block, inserting the gameMode var into the part where it changes. Given the length of the return statement it actually reduces quite a bit of lines of code:

`var changeGameMode = function (input) {

  if (input == "lowest") {

    gameMode = "lowest";

    resetGame();

  } else if (input == "highest") {

    gameMode = "highest";

    resetGame();

  }

    return "

    Initiating new game rules: the player with the <b>${gameMode}</b> combined number wins! <br><br>

    The score has been reset.<br><br>

    Player 1, start by hitting submit.

    ";

};`